src/components/Pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { Box, Button, TextField, Typography, Link, Grid, Paper, Avatar } from '@mui/material';
+import { styled } from '@mui/system';
+import background from '../../../assets/background.png'; // Importe a imagem de fundo
```

Comment:
Remover comentátrios
```suggestion
import background from '../../../assets/background.png'; 
```